src/foundry/foundry.js/textureUtils.d.ts

```diff
@@ -13,16 +13,13 @@ declare function getTexture(src: string): PIXI.Texture | null;
 /**
  * Load a single texture and return a Promise which resolves once the texture is ready to use
  * @param src      - The requested texture source
+ * @param options  - Additional options which modify texture loading
  * @param fallback - A fallback texture to use if the requested source is unavailable or invalid
  */
 declare function loadTexture(
   src: string,
-  {
-    fallback
-  }?: {
-    /**
-     * A fallback texture to use if the requested source is unavailable or invalid
-     */
+  options?: {
+    /** A fallback texture to use if the requested source is unavailable or invalid */
     fallback?: string;
```

Comment:
I would suggest to explicitly allow `undefined`, too (for strictOptionalTypes)

```suggestion
    fallback?: string | undefined;
```